src/utils/onlineParser.js

```diff
@@ -13,62 +13,47 @@
  *
  * Options:
  *
+ *     eatWhitespace: (stream: Stream | GraphQLStream) => boolean
```

Comment:
`CodeMirrorStream | GraphQLStream` ? `Stream` often means something specific that's different from this